src/PosixSemaphore.php

```diff
@@ -15,37 +15,39 @@
 final class PosixSemaphore implements Semaphore
 {
     private const LATENCY_TIMEOUT = 0.01;
+    private const MAX_ID = 0xffffffff;
```

Comment:
We should probably use the signed int maximum instead of the unsigned int maxium?

Reply:
Not sure it matters, any non-zero value works locally.

Reply:
Let's see whether it'll be an issue.